lib/pilot/build_manager.rb

```diff
@@ -17,7 +17,7 @@ def upload(options)
       if result
         Helper.log.info "Successfully uploaded the new binary to iTunes Connect"
 
-        unless config[:skip_submission]
+        unless config[:skip_submission] && config[:changelog].nil?
```

Comment:
It's confusing to have 2 statements in one `unless`. Could you change it be "more positive"?